README.md

```diff
@@ -26,6 +26,8 @@ Note: none of your data is modified nor sent anywhere during execution
 # Dependencies easy install
 If you don't have pip, see https://pip.pypa.io/en/stable/installing/
 
+Another option on Mac OS X is to use `sudo easy_install pip`
```

Comment:
If you don't have pip, see https://pip.pypa.io/en/stable/installing/, or if using a Mac run `sudo easy_install pip`